Orange/tests/test_datasets.py

```diff
@@ -5,22 +5,23 @@
 
 import os
 
-import Orange
+from Orange.data import Table, Variable
+from Orange import datasets
 
 
 class TestDatasets(unittest.TestCase):
     def test_access(self):
-        d1 = Orange.datasets.anneal
-        fname = Orange.datasets.anneal['location']
-        d2 = Orange.datasets['anneal']
+        d1 = datasets.anneal
+        fname = datasets.anneal['location']
+        d2 = datasets['anneal']
         self.assertNotEqual(len(d1), 0)
         self.assertEqual(len(d1), len(d2))
 
     def test_filter(self):
-        discr = [info for info in Orange.datasets.values()
+        discr = [info for info in datasets.values()
                  if info['features']['continuous'] == 0]
         for info in discr:
```

Comment:
Since `discr` isn't used anywhere else, you could easily merge these two iterations into one (this) `for` and one `if` beneath it.